lxd/node/config.go

```diff
@@ -169,16 +168,53 @@ var ConfigSchema = config.Schema{
 	"storage.images_volume":  {},
 }
 
+func validateCoreHTTPSAddress(value string) error {
+	if value == "" {
+		return nil // Deleting entry
+	}
+
+	host := value
+	splitHost, _, err := net.SplitHostPort(value)
+	if err == nil {
+		host = splitHost
+	}
+
+	ip := net.ParseIP(host)
+	if ip != nil {
+		return nil
+	}
+
+	_, err = net.LookupHost(host)
+	if err == nil {
+		return nil
+	}
+
+	return fmt.Errorf("Only IPV4, IPV6 address, :<HOST>:<PORT>, or a valid and resolvable Hostname are valid")
```

Comment:
Is that first colon in the error message a mistake, isn't it the same form as in `validateClusterHTTPSAddress`, i.e `<HOST>:<PORT>` not `:<HOST>:<PORT>`?

Reply:
I guess it's either a typo or a mix of things. It's getting hard to cover everything that's supported in the error message though. What's supported is:
 - Wildcard addresses without a port (`0.0.0.0`, `::`, `[::]`)
 - Wildcard addresses with a port (`0.0.0.0:8443`, `[::]:8443`)
 - IPv4/IPv6 address without a port (`1.2.3.4`, `1234:4567`, `[1234::4567]`)
 - IPv4/IPv6 address with a port (`1.2.3.4:8443`, `[1234::4567]:8443`)
 - DNS record without a port (`blah`)
 - DNS record with a port (`blah:8443`)
 - Port only (`:8443`)

Reply:
So we probably ought to check the validator against all of those ^ but the error itself should probably just be `Bad or unavailable listen address: %s'

Reply:
Yeah makes sense, the inconsistency just stood out as a warning sign to me :)